.rubocop_todo.yml

```diff
@@ -32,8 +32,6 @@ Lint/AmbiguousRegexpLiteral:
 # Configuration parameters: AllowSafeAssignment.
 Lint/AssignmentInCondition:
   Exclude:
-    - 'lib/riak/bucket_type.rb'
```

Comment:
Because it would be an enormous task to un-exclude and fix them. Ideally we'd remove one file at a time and do a PR on the fixes.